virttest/utils_conn.py

```diff
@@ -1545,13 +1617,37 @@ def conn_setup(self):
                             'unix_sock_rw_perms="%s"' % unix_sock_rw_perms}
             self.client_libvirtdconf.sub_else_add(pattern2repl)
 
+        # edit the /etc/sasl2/libvirt.conf to change sasl method
```

Comment:
it sound like it also update /etc/hosts, so please add this information too in your comment